pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -457,7 +457,123 @@ func Test_server_Search(t *testing.T) {
 		}
 		return nil
 	}
+
+	/*
+		Search test cases:
+		- Equivalence Class Testing
+			- uint8, float32 common
+				- case 1: vector success
+				- case 2: different dimension fail
+		- Boundary Value Testing
+		  	- uint8, float32 common
+				- case 1: min value vector success
+				- case 2: max value vector success
+				- case 3: 0 length vector fail?
+				- case 4: max length vector fail?
+				- case 5: nil vector fail
+			- uint8
+			- float32
+				- case 6: 0 value vector success
```

Comment:
I think uint8 also need 0 value test

Reply:
I intentionally excepted uint8 0 value test because it can be from 0 to 255.

Reply:
so is it the same test case as the following test case?
```
		  	- uint8, float32 common
				- case 1: min value vector success
```

Reply:
Yes, that's right.

Reply:
IC, thank you :)